### PR TITLE
TW-1135: preserve users linebreaks in message content

### PR DESCRIPTION
**Pull request by imGok**
https://github.com/linagora/flutter_matrix_html/pull/12
https://github.com/linagora/matrix-dart-sdk/pull/42

# Demo

https://github.com/linagora/twake-on-matrix/assets/48354990/7f840ef7-91ce-40ad-a17f-fb191f0035b3

# Linebreaks proofs

image
image

#1135

### CODE REVIEW

File: pubspec.lock [hunk 1]
```diff
@@ -1567,7 +1567,7 @@ packages:
     description:
       path: "."
       ref: "twake-supported-0.22.6"
-      resolved-ref: bfff7db647b177a0a0e04780e063bfbdd22031b8
+      resolved-ref: "239609368f4278e4711e28949738f513b1a8bbcf"
```
Comment (hoangdat) [inline pubspec.lock:1570]: you should only use: `e5e67b2d0158b9398eb6430841c2bed7f61d8997` to `main`.

**Review by sherlockvn — DISMISSED** [review]

**Comment by hoangdat** [comment]
I tested, it not work well, please show the demo.

**Comment by imGok** [comment]
@hoangdat I confirm it's working on my side after cleaning all libs etc. the deployment is not updated

**Comment by sherlockvn** [comment]
Work from my side 
image

**Comment by sherlockvn** [comment]
Error when i try: 
"""
adsasd
adsasdd
"""
result:
image
expected:
"""
adsasd
adsasdd
"""

**Comment by imGok** [comment]
> Error when i try: 
> 
> """
> 
> adsasd
> 
> adsasdd
> 
> """
> 
> result:
> 
> image
> 
> expected:
> 
> """
> 
> adsasd
> 
> adsasdd
> 
> """

It's because we trim the string, it's intentional. @hoangdat do we remove it ?

**Comment by imGok** [comment]
@sherlockvn 
Please check again on your local version (pr deployment is not updated)
https://github.com/linagora/twake-on-matrix/assets/48354990/16d71dbd-4234-40a7-ab53-2970308f9e76

**Comment by sherlockvn** [comment]
Okay i can confirm it's work from my side too
image

**Pull request merged**